rooms.js

```diff
@@ -933,16 +933,34 @@ class BattleRoom extends Room {
 	}
 	requestKickInactive(user, force) {
 		if (this.resetTimer) {
-			if (user) this.sendUser(user, '|inactive|The inactivity timer is already counting down.');
-			return false;
+			let timerSetByUser = user && (this.resetUser === user.userid || (user in this.game.players && this.resetUser === '+'));
```

Comment:
Wait, this is still in `rooms.js`. Why hasn't anyone refactored the timer to roomgame like I asked? -_-